input/resources/Library-FHIR-ModelInfo.json

```diff
@@ -13,6 +13,11 @@
   }, {
     "url": "http://hl7.org/fhir/uv/cpg/StructureDefinition/cpg-knowledgeRepresentationLevel",
     "valueCode": "structured"
+  }, {
+    "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-softwaresystem",
+    "valueReference": {
+      "reference": "http://hl7.org/fhir/us/cqfmeasures/STU3/Device-software-system-example.html"
+    }
   } ],
```

Comment:
For the CQF, I just removed all the cqf-tooling stuff, so we shouldn't add this back in